### PR TITLE
Creation of a CONTRIBUTING.md file (EZP-27167)

### PR DESCRIPTION
**Pull request by mere-teresa**
I tried to avoid redundancy with the online version while giving all the necessary informations to an offline reader.

### CODE REVIEW

**Comment by mere-teresa** [comment]
https://jira.ez.no/browse/EZP-27167

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+Are you ready to become a part of the eZ Community? There are several ways in which you can contribute, from spotting and reporting bugs to commenting on the documentation to discussing innovative uses on Slack to coding new bundles.
+We have a [Bug Tracker](https://jira.ez.no) and you can talk to a human on eZ Community Slack: https://ezcommunity.slack.com
+
+
+# Development Guidelines
+
+Our Development Guidelines are available on this page: https://doc.ez.no/display/DEVELOPER/Development+guidelines
+
+## Topics covered by this Guidelines
```
Comment (DominikaK) [inline CONTRIBUTING.md]: I'm not sure it makes sense to list these points here (unless you add links for each point). To my mind the general link to the Confluence doc is enough.

Comment (mere-teresa) [reply CONTRIBUTING.md]: I was thinking: you are offline, and you have to make the decision "_do you really need the Guidelines ?_"

Comment (DominikaK) [reply CONTRIBUTING.md]: You still can't make the decision without the details, can you?
But if you're sure, no problem with leaving it, if so just remember to change `this Guidelines` to `the Guidelines`

**Pull request merged**